lib/tasks/journeys_report.rake

```diff
@@ -97,24 +49,35 @@ namespace :journeys do
           from: move.from_location.title,
           to: move.to_location&.title,
           events: move.move_events.default_order.map do |move_event|
-            {
-              timestamp: move_event.client_timestamp.strftime('%d/%m %H:%M:%S'),
-              event: move_event.event_name,
-              notes: move_event.notes,
-              from_location: move_event.from_location&.title,
-              to_location: move_event.to_location&.title,
-            }
-          end,
+                    {
+                      timestamp: move_event.client_timestamp.strftime('%d/%m %H:%M:%S'),
+                      event: move_event.event_name,
+                      notes: move_event.notes,
+                      from_location: move_event.from_location&.title,
+                      to_location: move_event.to_location&.title,
+                    }
+                  end,
           journeys: journeys,
         }
       end
       suppliers << {
         supplier: supplier.key,
         moves: moves,
       }
-    end
-    File.open('journeys_report.json', 'w') do |file|
-      file.write(JSON.pretty_generate({ data: suppliers }))
+
+      File.open(filename, 'w') do |file|
+        file.write(JSON.pretty_generate({ data: suppliers })) # maybe change to .to_json for production
```

Comment:
```ruby
data = {
  data: {
    supplier: supplier.key,
    moves: moves,
  }
}

json_data = JSON.pretty_generate(data)

File.write(filename, json_data)
```

Reply:
Feels a bit cleaner with better labels for things in lexical scope

Reply:
I moved the supplier key to the "header" of the moves object.

```
  "moves": [
    {
      "supplier": "geoamey",
      "reference": "AHC5219N",
      "date": "2020-06-13",
      "from": "Medway STC",
      "to": "Colchester Crown Court",
      "events": [

      ],
      "journeys": [
        [....]
```
as I was thinking about the case where multiple files are just merged together